charts/openshift-metering/templates/presto/presto-common-config.yaml

```diff
@@ -53,6 +53,9 @@ data:
       echo "Adding hive.s3.aws-access-key and hive.s3.aws-secret-key to $HIVE_CATALOG_CONFIG"
       echo "hive.s3.aws-access-key=$AWS_ACCESS_KEY_ID" >> "$HIVE_CATALOG_CONFIG"
       echo "hive.s3.aws-secret-key=$AWS_SECRET_ACCESS_KEY" >> "$HIVE_CATALOG_CONFIG"
+      echo "hive.s3.endpoint={{ .Values.presto.spec.config.s3Compatible.endpoint }}" >> "$HIVE_CATALOG_CONFIG"
```

Comment:
This new section of stuff is going to need a condition around it so it's only set when s3Compatible is used

Reply:
Actually, all of this can go straight into the hive.properties template: https://github.com/operator-framework/operator-metering/blob/master/charts/openshift-metering/templates/presto/_presto_helpers.tpl#L1-L25